packages/cfd/src/Components/cfd-real-account-display.tsx

```diff
@@ -131,7 +129,7 @@ const CFDRealAccountDisplay = ({
         if (citizen && tax_residence) {
             const is_tin_required = landing_companies?.config?.tax_details_required ?? false;
             return (
-                is_tin_required ||
+                !is_tin_required ||
```

Comment:
This change makes has_required_credentials be true when citizen && tax_residence are filled out but tax id is not in case it's not required (when tax_details_required equals 0). It has to be fixed in this PR, otherwise a test covering this case will fail.